controllers/api_controller.js

```diff
@@ -48,4 +48,29 @@ module.exports.about = (req, res) => {
         if(err) return res.status(404).json({message: 'error', story: ''});
         return res.json({story, message: 'success'});
     });
+}
+
+module.exports.ideas = (req, res) => {
```

Comment:
The code looks good and decent. Have you tested this API with postman?

Reply:
Yes I have tested with postman and it worked fine.